src/forms/StaticField.stories.tsx

```diff
@@ -15,6 +16,9 @@ export function StaticField() {
         <StaticFieldComponent label="First" value="Bob" />
         <StaticFieldComponent label="First" value="Bob" />
       </FieldGroup>
+      <StaticFieldComponent label="First">
+        <Pills values={["First", "Last"]} />
```

Comment:
@Chieze-Franklin FWIW this PR let's `StaticField` have children that are not "just a string".